docs/scaffold-a-new-project/README.md

```diff
@@ -0,0 +1,72 @@
+# Scaffold a New Project
+
+Neutrino provides a scaffolding tool to help you quickly jump-start a project. `create-neutrino-project` uses
+presets behind the scene to build projects. If you are not familiarized with them, take a moment to
+explore [what presets are](https://neutrino.js.org/presets/).
+
+## Getting Started
+Run the following command to start the interactive session. Substitute `<directory-name>` with the directory name you wish to create for this project.
+
+### Yarn
+
+```
+❯ yarn create neutrino-project <directory-name>
+```
+
+_Note: The `create` command is a shorthand that helps you do two things at once. See the [Yarn create docs](https://yarnpkg.com/lang/en/docs/cli/create/) for more details._
+
+### npm
+
+```
+❯ npm install -g create-neutrino-project
+❯ create-neutrino-project <directory-name>
+```
+
+## Available Projects
+`create-neutrino-project` presently offers a scaffolding project to build an application, a library,
+or components. Depending on the project type, the CLI helper may offer different flavors
+of that project to scaffold. Each project type harnesses the power of a preset to configure itself.
+
+| Project     | Project Type     | Preset                                                                                                             |
```

Comment:
To make this way easier, you only need a single space between dividers, and the row divider only needs 3 dashes:

```md
| Project | Project Type | Middleware |
| --- | --- | --- |
```

Also, invert the project and project type values. It is a "React project", which is an "Application project type".

Reply:
I used a [markdown table generator](https://www.tablesgenerator.com/markdown_tables) to make these tables. It works great. You still want me to put 3 dashes instead?